pkg/component/clusterautoscaler/cluster_autoscaler_test.go

```diff
@@ -89,6 +89,8 @@ var _ = Describe("ClusterAutoscaler", func() {
 		configScanInterval                        = &metav1.Duration{Duration: time.Second}
 		configIgnoreDaemonsetsUtilization   bool  = true
 		configVerbosity                     int32 = 4
+		configMaxEmptyBulkDelete                  = pointer.Int32(10)
```

Comment:
Maybe use a different value here rather than the default?

Reply:
Done